src/components/map/Map.js

```diff
@@ -36,14 +35,17 @@ const Map = ({
     bootstrapURLKeys={bootstrapURLKeys}
     options={() => ({
       mapTypeId: mapType,
+      disableDefaultUI: true,
+      // TODO: should we disable tilt?
+      // tilt: 0,
     })}
     layerTypes={layerTypes}
     center={view.center}
     zoom={view.zoom}
     onChange={onViewChange}
     resetBoundsOnResize
   >
-    {!geolocation.loading && (
+    {geolocation && !geolocation?.loading && (
```

Comment:
Can we get rid of the operational chaining on the second condition? 

`geolocation` will never be null nor undefined since we short-circuit on the first condition.